### PR TITLE
20230817 idv migration

### PR DESCRIPTION
**Pull request by Firstyear**
Tidy up some parts of IDV now that it's merged. Generally this makes key gen simpler, and fixes the missing migration that was needed to trigger key gen on existing accounts.

- [ x ] This pr contains no AI generated code
- [ x ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ x ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: server/lib/src/plugins/eckeygen.rs [hunk 1]
```diff
@@ -175,6 +120,9 @@ mod tests {
         );
     }
 
+    /*
+    // Invalid, can't be set due to no impl from clone_value
```
Comment (Seba-T) [inline server/lib/src/plugins/eckeygen.rs:124]: Huh so before it was working but for a different reason than I thought?

**Comment by Seba-T** [comment]
Everything seems good to go to me :)

**Review by yaleman — APPROVED** [review]
I guess that explains why my PR was so broken when I rebased it this morning 😡

**Comment by Seba-T** [comment]
> I guess that explains why my PR was so broken when I rebased it this morning 😡 

Wopsies 😵

**Pull request merged**